src/current/cockroachcloud/folders.md

```diff
@@ -5,27 +5,25 @@ toc: true
+
 {{site.data.alerts.callout_success}}
-{% include_cached feature-phases/limited-access.md %}
+{% include_cached feature-phases/preview.md %}
 {{site.data.alerts.end}}
 
-This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters with folders using the CockroachDB {{ site.data.products.cloud }} Console. You can also use the [CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}), or the [Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach) v1.1.0 or above. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}]({% link cockroachcloud/managing-access.md %}).
-
 ## How folders work
 
 Folders allow you to organize and manage access to your clusters according to your organization's requirements. For example, you can create top-level folders for each business unit in your organization, and within those folders, you can organize clusters by geographic location and then by their level of maturity, such as production, staging, and testing. For more details, refer to [Folder Structure](#folder-structure).
 
 Each folder can contain a mix of folders and clusters. Roles assigned on a folder are inherited by its descendants.
 
-Each folder that you create has an ID. To create a cluster in a folder or to move a cluster into a folder, you set its `parent_id` field to the folder's ID.
+Each folder that you create has an ID, which determines its position in the organization's hierarchy.
 
-If no parent ID is specified for a cluster, it is created at the root level of the organization. Clusters and folders at the organization level have their `parent_id` set to `root`.
+- Clusters or folders within a folder have their parent ID set to that folder's ID.
+- If a cluster or folder has no parent ID, or if its parent ID is set to `root`, its location is the root level of the organization.
+- Moving a folder is a lightweight operation that does not modify the folder itself or its descendant folders or clusters; you need only update the folder's parent ID. Similarly, to move a cluster into or out of a folder, you need only update the cluster's folder ID. If you move a folder that contains descendant resources, the descendant resources are not directly modified.
```

Comment:
The details about updating the parentID seem more relevant to an API specific implementation.  From the perspective of the UI, you would just use the move modals.   I can see value in describing both and calling out the differences.